openmc/filter.py

```diff
@@ -1332,6 +1332,35 @@ def check_bins(self, bins):
             cv.check_greater_than('filter value', v0, 0., equality=True)
             cv.check_greater_than('filter value', v1, 0., equality=True)
 
+    def get_tabular(self, values, interpolation='histogram'):
+        """Creates a openmc.stats.Tabular distribution using the EnergyFilter
+        bins and the provided values. Intended use is to help convert a
+        spectrum tally into a source energy.
```

Comment:
```suggestion
        """Create a tabulated distribution based on tally results with an energy filter

        This method provides an easy way to create a distribution in energy
        (e.g., a source spectrum) based on tally results that were obtained from
        using an :class:`~openmc.EnergyFilter`.
```

Reply:
thanks, i didn't know about that `~ ` syntax.